youtube_dl/YoutubeDL.py

```diff
@@ -833,6 +833,8 @@ def iter_playlistitems(format):
                                                       extra_info=extra)
                 playlist_results.append(entry_result)
             ie_result['entries'] = playlist_results
+            if result_type == 'playlist':
```

Comment:
because I don't know what is 'multi_video', and this would apply to 'multi_video' without this condition. Do I have to remove it?